examples/05_glm_second_level/plot_second_level_one_sample_test.py

```diff
@@ -152,7 +152,12 @@
 # This threshold is much more conservative than an uncorrected threshold, but
 # is still more liberal than a typical corrected threshold for this kind of
 # analysis, which tends to be ~0.05.
-threshold = 1  # p<0.1
+#
+# We will also cap the negative log10 p-values at 2.7, because this is the
+# maximum observable value for the nonparametric tests, which were run with
+# only 500 permutations.
+threshold = 1  # p < 0.1
+vmax = 2.7  # ~= -np.log10(1 / 500)
```

Comment:
Actually should be slightly below (2.69).

Reply:
Done in [1e7bc40](https://github.com/nilearn/nilearn/pull/3181/commits/1e7bc40628b701735bd58b97082e1ba6493d314a). Thanks!